docs/usage.md

```diff
@@ -64,12 +64,35 @@ Options:
   --help                          Show this message and exit.
 ```
 
+
+## `watchmaker` as a standalone package
+
+Once a Watchmaker standalone executable has been
+[downloaded](installation.html) and a
+[configuration file](configuration.html) has been created (or you have decided
+to use the default configuration), using Watchmaker is the same as
+using Watchmaker as a CLI utility except that you run the downloaded file.
+
+For example, on Linux, you can view the CLI options (shown above) using
+the same flag as above.
+
+```shell
+# ./watchmaker-latest-standalone-linux-x86_64 --help
+```
+
+From Windows, execute Watchmaker by double-clicking on the
```

Comment:
Let's not encourage anyone to double-click on the stand-alone bin... that'll immediately start executing, which will only ever run the default config...